src/java.desktop/share/classes/javax/swing/JScrollBar.java

```diff
@@ -780,6 +780,7 @@ public void setMaximumSize(Dimension maximumSize) {
     }
 
     /**
+     * Returns the minimum size for the {@code JScrollBar}.
      * The scrollbar is flexible along its scrolling axis and
      * rigid along the other axis.
      * As specified in {@code setMinimumSize} JScrollBar will derive the
```

Comment:
This doesn't look good to me. It is `getMinimumSize` that specifies what is returned; `setMinimumSize` just warns the programmer that `getMinimumSize` ignores the value passed to the `set-` method.